release-notes/changes-in-timescaledb-2.md

```diff
@@ -320,6 +320,40 @@ After upgrading to TimescaleDB 2.0, **retention policies will no longer fail due
 continuous aggregates** and users have to ensure that retention and continuous aggregate policies have the 
 desired interplay. 
 
+Another change in 2.0 is that `drop_chunks` and the retention policy do not refresh continuous 
+aggregates on the dropped chunks. Previously dropping a chunk would call a refresh on each 
+continuous aggregate to recompute it according to existing invalidations. The refresh was 
+unlikely to include all updates due to `ignore_invalidations_older_than`, which prevents to 
+record invalidations for updates older than the threshold. If a user carefully crafted 
+parameters to the retention policy and the continuous aggregate that no update is missed, 
+the refresh could include previously dropped values leading to unexpected outliers in 
+aggregated values. TimescaleDB 2.0 introduces new function 
+`_timescaledb_internal.refresh_continuous_aggregate`, which refreshes 
+exactly a chunk, and can be used together with drop_chunks. For example, following action can 
+be defined:
+
+```sql
+CREATE OR REPLACE PROCEDURE refresh_and_drop_policy(job_id int, config jsonb) LANGUAGE PLPGSQL AS
+$$
+DECLARE
+  drop_after interval;
+  hypertable text;
+BEGIN
+  SELECT jsonb_object_field_text(config, 'drop_after')::interval
+    INTO STRICT drop_after;
+  SELECT jsonb_object_field_text(config, 'hypertable')
+    INTO STRICT hypertable;
+  BEGIN
+    SELECT refresh_chunks(cagg, show_chunks(older_than => drop_after))
+    FROM _timescaledb_catalog.continuous_agg cagg, _timescaledb_catalog.hypertable hyper
+    WHERE hyper.id = cagg.raw_hypertable_id AND 
+      hyper.schema_name || '.' || hyper.table_name = hypertable;
+    SELECT drop_chunks(hypertable, older_than => drop_after);
+  END;
+END
+$$;
+```
+
```

Comment:
The above user-defined action can be scheduled as a custom policy using the `add_job` function.

Reply:
@erimatnor Will it be good to add call to `add_job` in the script, like in the design doc?

Reply:
I guess it is fine, if we expect users to run this always via policy, as opposed to manually.

Reply:
> The above user-defined action can be scheduled as a custom policy using the `add_job` function.

I interpret it as it is good to show adding the policy with `add_job` in the example, so it is added.